factory/processing/blockProcessorCreator.go

```diff
@@ -1116,6 +1116,11 @@ func (pcf *processComponentsFactory) createMetaTxSimulatorProcessor(
 		return nil, err
 	}
 
+	err = builtInFuncFactory.SetPayableHandler(vmFactory.BlockChainHookImpl())
```

Comment:
I think this is already fixed in the PR that changes the tx simulation construction. You should check at merge time when resolving the (future) conflict :D

Reply:
fixed